lib/board.rb

```diff
@@ -0,0 +1,20 @@
+class Board
+  attr_reader :cells
+  def initialize
+    @cells = Hash.new(0)
+    build_board 
+  end
+
+  def build_board
+    ('A'..'D').each do |letter|
+      (1..4).each do |number|
+        coordinate = "#{letter}#{number}"
+        cell_default = Cell.new(coordinate)
```

Comment:
I'm not sure if we need to pass in the coordinate here as an argument since it should just make our coordinates with the enumerable